### DISCUSSION TITLE
Slack & Discord & Github 🤯 Where should I post my question/bug report/feature request etc?

### DISCUSSION THREAD

**Discussion by stichbury** [kedro-org/kedro#1986] [Announcements]
Hello world!

This is a post to describe how we can work together as a community to optimise the turnaround time for questions and bug reports.

Going forward, from December 2022, the main online channels for the Kedro community will be our Slack org and this Github project, since the Discord Server will move to read only, as it [**heads off into the sunset, on November 30th 2022**.](https://discord.com/channels/778216384475693066/778996598880862208/1029694324079865916)

## What's happening to Discord?
We've been struggling to grow our community on Discord because a large number of Kedroids cannot use it (it's banned in their organisations). So we've decided to rebuild our community in Slack because it seems to be the most widely accepted. and used by other communities such as Great Expectations, dbt and MLflow. 

You can already see a [searchable archive on Linen](https://linen-discord.kedro.org/) that contains of all conversations that have happened on Discord. This will be kept for future reference. 

**Update**: We also have a [searchable archive of Slack conversations](https://linen-slack.kedro.org/) for all the conversations that have happened since the community moved onto it.

## Move onto Slack
Now's the time to join and use the [Kedro Slack organisation](https://kedro.slack.org). 

We know that it's inconvenient so we're sweetening the deal with [a massive Kedro swag drop --join in here](https://www.surveys.online/jfe/form/SV_8jfTn7SQDcUiN5c)!

## What to post where?
You want answers. We want you to get answers. So here's how.

* ❓ Do you have a question about your project? Something not working like the docs says it should? Or you can't find any docs at all? You want to try something a bit different to what the examples do?
    * You need the community. They can best answer you. Ask your question over on the [#questions channel of the Slack org](https://kedro-org.slack.com/archives/C03RKP2LW64). We'll see you over there! 🏃 💨 
    
* 🐞 Have you found a bug? It's worth running it past the community on the [#questions channel of the Slack org](https://kedro-org.slack.com/archives/C03RKP2LW64) to see if anyone else has seen it, experienced it and realised it's a Kedro feature that we've not explained well. 
    * If you're sure it's a bug (or it's a badly worded feature) then please post it as an [issue here on the Github repo](https://github.com/kedro-org/kedro/issues)
 
* 🤔 Do you have an idea about where the Kedro project should go next? Do you have a feature request? Do you need to poll the community to find out what other Kedroids think?
    * That's the time to create discussion here on the [Github discussions section](https://github.com/kedro-org/kedro/discussions) 

* 🎨 Do you have something to show us all? 
    * Again, here's the place -- use the [Show and tell section of Github discussions](https://github.com/kedro-org/kedro/discussions). We 🖤 seeing your blog posts, examples and plugins.

## In summary
Discord is closing. Instead, join [Slack](https://kedro.slack.org) and [get swag](https://www.surveys.online/jfe/form/SV_8jfTn7SQDcUiN5c).

### Got something to say? 

* If it's a question, use [Slack](https://kedro.slack.org). 
* If it's a bug, use [Github issues](https://github.com/kedro-org/kedro/issues). 
* If it's to start a community conversation, use [Github discussions](https://github.com/kedro-org/kedro/discussions).

Not sure? Start over on Slack. There are more community members on there regularly, ready to help direct you to the fastest way to get what you need.

![giphy (2)](https://user-images.githubusercontent.com/5180475/198307809-5eff0f0b-e2a1-46c0-ac30-bfdab0368bfa.gif)

**Comment by lineshFT** [comment]
Slack link doesn't seem to work!

**Reply by astrojuanlu** [reply]
We're fixing it! In the meantime please use https://join.slack.com/t/kedro-org/shared_invite/zt-265nr2u1i-kXI55~tRkp91mxBupL8FMQ (⚠️ if you read this in the future, the link might not work anymore)
